StackExchange.Redis.Resilience/StackExchange.Redis.Resilience.csproj

```diff
@@ -3,11 +3,11 @@
   <PropertyGroup>
     <Product>StackExchange.Redis.Resilience</Product>
     <Title>StackExchange.Redis.Resilience</Title>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
     <Authors>maca88</Authors>
     <Company />
     <Description>Provides a solution to force reconnect when StackExchange.Redis fails to reconnect after a connection blip.</Description>
-    <TargetFrameworks>net472;net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;net461;netstandard2.0;net462</TargetFrameworks>
```

Comment:
Why did you add `net462`? I think it would make more sense to align it with [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis/blob/main/src/StackExchange.Redis/StackExchange.Redis.csproj#L4) `TargetFrameworks`.

Reply:
You are absolutly right, fixed.